bindings/java/src/main/native/Makefile.am

```diff
@@ -0,0 +1,48 @@
+JDK = $(JAVA_HOME)
+topdir=../../../../..
+javadir=$(topdir)/bindings/java
+MVN=@cd $(javadir) && mvn
+ucm_lib=ucm/.libs
+ucs_lib=ucs/.libs
+uct_lib=uct/.libs
+ucp_lib=ucp/.libs
+sources=bridge.cc request_util.cc
+
+# additional include paths necessary to compile the C library
+AM_CPPFLAGS = -fPIC -DPIC -Werror \
+		-I$(JDK)/include -I$(JDK)/include/linux \
+		-I$(topdir)/src/
+
+AM_CXXFLAGS = -std=c++11
```

Comment:
why?

Reply:
Currently not needed, will be in use in the next commits. Deleting it for now.